adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/routes/AssetsResource.kt

```diff
@@ -167,6 +155,96 @@ class AssetsResource {
         return getPage(pageSize, page, releases)
     }
 
+    @GET
+    @Path("/release_name/{vendor}/{release_name}")
+    @Operation(
+        summary = "Returns release information",
+        description = "List of releases with the given release name"
+    )
+    @APIResponses(
+        value = [
+            APIResponse(
+                responseCode = "200", description = "Release with the given vendor and name"
+            ),
+            APIResponse(responseCode = "400", description = "bad input parameter"),
+            APIResponse(responseCode = "404", description = "no releases match the request"),
+            APIResponse(responseCode = "404", description = "multiple releases match the request")
```

Comment:
Just wondering if there's a better HTTP code for this scenario, 404 suggests it's the clients fault as they've requested a release that doesn't exist but that's not true and there's nothing they can do to fix it. 

It feels like a constraint/problem of the API, and maybe something we'll resolve in the future.

Maybe this HTTP 501 fits better?

> 501 Not Implemented
> The server either does not recognize the request method, or it lacks the ability to fulfil the request. Usually this implies future availability (e.g., a new feature of a web-service API).[64]

Reply:
yeah I did go around a bit on what would be the correct behaviour here, some sort of 500 error is not totally unreasonable as it would mean that adopt have messed up and created 2 releases with the same name

Reply:
ah ok, yeah so maybe just a straight up HTTP 500 makes sense to indicate the invalid state if we'd never expect or want this to happen